flowforge/input/System.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, components: Dict[str, Component], sysdict: Dict, unitdict: Di
         self._MMBC = None
         self._EBC = None
         self._fluid = None
+        self._gasfluid = None
```

Comment:
@bscollin, I don't really like having `fluid` and `gasfluid`.  Part of me just thinks `fluid` and `void`.

Reply:
Using fluid and gas